infra/cifuzz/fuzz_target.py

```diff
@@ -42,10 +42,9 @@
 BUFFER_TIME = 10
 
 # Log message if we can't check if crash reproduces on an recent build.
-COULD_NOT_TEST_ON_RECENT_MESSAGE = (
-    'Crash is reproducible. Could not run recent build of '
-    'target to determine if this code change (pr/commit) introduced crash. '
-    'Assuming this code change introduced crash.')
+COULD_NOT_TEST_ON_CLUSTERFUZZ_MESSAGE = (
+    'Could not run ClusterFuzz (old) build of target to determine if this '
+    'code change (pr/commit) introduced crash. Assuming crash is novel.')
```

Comment:
Is "novel" a term that people are familiar with outside of academia? Might be better to use simpler wording here, like "crash has not been seen before".